recipe/meta.yaml

```diff
@@ -9,11 +9,9 @@ package:
 source:
   url: https://github.com/Galsim-Developers/galsim/archive/v{{ version }}.tar.gz
   sha256: {{ sha256 }}
-  patches:
-    - 0001-remove-run-install-checks.patch
```

Comment:
There used to be a patch adding the line 
```
cpp = os.environ.get('CXX', cpp)
```
which honestly sounds like something I should add upstream, but I never knew this was needed here.  But that might be required again I guess as a patch, since that line still isn't in the main-line setup.py file.

cf. https://github.com/conda-forge/galsim-feedstock/blob/ad911abac5b59353c2499642deca884f3b35cb74/recipe/setup.py.patch

Reply:
Update, if you find you need this, I added that line to the upstream setup.py, but using the environment variable GALSIM_CXX, since I wasn't sure that taking in a CXX in the user env would always be appropriate.  But the functionality exists to explicitly specify the compiler if you want using this syntax:
```
GALSIM_CXX=clang++ pip install -vv .
```
In case you need it.